### PR TITLE
Fix error on user create.

### PR DESCRIPTION
**Pull request by ovista-dept**

### CODE REVIEW

File: web/modules/custom/webdir_department_manager/src/Plugin/field_group/FieldGroupFormatter/Relationships.php [hunk 1]
```diff
@@ -32,6 +32,11 @@ public function process(&$element, $processed_object) {
     parent::process($element, $processed_object);
     
     // Get the user ID
+    $user = \Drupal::routeMatch()->getParameter('user');
```
Comment (mlsamuelson) [inline web/modules/custom/webdir_department_manager/src/Plugin/field_group/FieldGroupFormatter/Relationships.php:35]: New code here should be above this comment, not below it. Please fix.

**Review by mlsamuelson — CHANGES_REQUESTED** [review]
See my comment on the code. Please fix.

File: web/modules/custom/webdir_department_manager/src/Plugin/field_group/FieldGroupFormatter/Relationships.php (hunk 1 above)
Comment (ovista-dept) [reply web/modules/custom/webdir_department_manager/src/Plugin/field_group/FieldGroupFormatter/Relationships.php:35]: I did a bit of refactoring, should be ok now.

**Comment by duarte-daniela** [comment]
@mlsamuelson I'm merging all the hostfixes into a single branch to have an environment for the demo and I'll send you a single PR.

**Pull request merged**